prebid-server/developers/add-new-bidder-go.md

```diff
@@ -504,7 +504,7 @@ if request.Imp[i].W == nil && request.Imp[i].H == nil && len(request.Imp[i].Form
 </details>
 <p></p>
 
-The second argument, `requestInfo`, is for extra information and helper methods provided by the core framework. For now, this just includes `requestInfo.PbsEntryPoint` which is commonly used to determine if the request is for AMP or Long Form Video Ad Pods. This object will be expanded in the future to also include currency conversion and extension unmarshalling helper methods.
+The second argument, `requestInfo`, is for extra information and helper methods provided by the core framework. For now, this includes `requestInfo.PbsEntryPoint` which is commonly used to determine if the request is for AMP or Long Form Video Ad Pods and the currency conversion helper method `ConvertCurrency(value float64, from, to string) (float64, error)`. This object will be expanded in the future to also include an extension unmarshalling helper method.
```

Comment:
I updated this too in my local copy. What do you think of:

The second argument, `requestInfo`, is for extra information and helper methods provided by the core framework. This includes:
- `requestInfo.PbsEntryPoint` to access the entry point of the bid request, commonly used to determine if the request is for AMP or for a Long Form Video Ad Pod.
- `requestInfo.GlobalPrivacyControlHeader` to read the value of the `Sec-GPC` Global Privacy Control (GPC) header of the bid request.
- `requestInfo.ConvertCurrency` a method to perform currency conversions.

Reply:
Much better and easier to read. Changed.